### DISCUSSION TITLE
What is the best practice for displaying lists in the frontend

### DISCUSSION THREAD

**Discussion by jptissot** [OrchardCMS/OrchardCore#10362] [Q&A]
I am trying to figure out the best way to display a list with paging in the frontend without using the ListPart. Do you guys have recommendations on how to do so? Should we add a feature that allow for easily list contents based on a query?

**Comment by Piedone** [comment]
I think you need to code it, even though you can use the built-in pager feature. But this is something I thought about having a feature too though I don't think we can do it without ultimately rebuilding the Projections feature of O1.

**Reply by Skrypt** [reply]
To be honest I never really liked Projections. Always ended up having my own code.
